$NetBSD: patch-aa,v 1.5 2013/10/10 17:16:54 roy Exp $

- configure paths for pkgsrc
- honor pkgsrc LDFLAGS
- DESTDIR support
- don't attempt to randomly blow things away on the system

--- Makefile.orig	2002-07-30 22:04:21.000000000 +0000
+++ Makefile
@@ -13,13 +13,14 @@ SYSDEF=NOTBSD
 
 # Prepend "c:" (or whatever drive you use) to the following paths for MSDOS
 # Location of high score file
-SFILE=/usr/games/lib/greed.hs
+SFILE=${PREFIX}/share/games/greed.hs
 # Location of game executable
-BIN=/usr/games
+BIN=${DESTDIR}${PREFIX}/bin
 
 # Flags for use with the Linux ncurses package (recommended)
-CFLAGS = -O -s -DNOTBSD
-TERMLIB = -lncurses
+#CFLAGS = -O -s -DNOTBSD
+CFLAGS+=	-DNOTBSD
+TERMLIB ?= -lncurses
 CC = gcc
 
 # Flags for use with stock curses
@@ -32,14 +33,14 @@ CC = gcc
 #TERMLIB = scurses.lib
 
 greed: greed.c
-	cc -DSCOREFILE=\"$(SFILE)\" -D$(SYSDEF) -DRELEASE=\"$(VERS)\" -o greed greed.c $(CFLAGS) $(TERMLIB)
+	cc -DSCOREFILE=\"$(SFILE)\" -D$(SYSDEF) -DRELEASE=\"$(VERS)\" -o greed greed.c $(CFLAGS) $(LDFLAGS) $(TERMLIB)
 
 greed.6: greed.xml
 	xmlto man greed.xml
 
-install: greed.6 uninstall
+install: greed.6
 	cp greed $(BIN)
-	cp greed.6 /usr/share/man/man6/greed.6
+	cp greed.6 ${DESTDIR}$(PREFIX)/$(PKGMANDIR)/man6
 
 uninstall:
 	rm -f $(BIN)/install /usr/share/man/man6/greed.6
